test/SILOptimizer/irgenprepare.sil

```diff
@@ -0,0 +1,31 @@
+// RUN: %target-sil-opt -module-name Swift -irgen-prepare %s -enable-sil-verify-all | %FileCheck %s
```

Comment:
@gottesmm I imagine this test isn't necessary anymore and we should instead be testing the IRGen phase?

This test is failing when I believe it used to pass before the stuff got moved to IRGen from IRGenPrepare.

Reply:
Yes.